doc/gitian-building.md

```diff
@@ -300,26 +339,26 @@ cd gitian-builder
 bin/make-base-vm --lxc --arch amd64 --suite precise
 ```
 
-There will be a lot of warnings printed during build of the image. These can be ignored.
+There will be a lot of warnings printed during the build of the image. These can be ignored.
 
 **Note**: When sudo asks for a password, enter the password for the user *debian* not for *root*.
 
 Getting and building the inputs
 --------------------------------
 
-Follow the instructions in [doc/release-process.md](release-process.md) in the bitcoin repository
-under 'Fetch and build inputs' to install sources which require manual intervention. Also follow
-the next step: 'Seed the Gitian sources cache', which will fetch all necessary source files allowing
-for gitian to work offline.
+Follow the instructions in [doc/release-process.md](release-process.md#fetch-and-build-inputs-first-time-or-when-dependency-versions-change) 
```

Comment:
Perhaps out of scope, but the "Register and download the Apple SDK" bit of the linked instructions is pretty confusing, given that we've said nothing so far about being within an OS X context.